1-JavaScript/Week2/homeworkSolition/ex6-sol-totalCost.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+/*------------------------------------------------------------------------------
+Full description at: https://github.com/HackYourFuture/Homework/tree/main/1-JavaScript/Week3#exercise-6-total-cost-is
+
+You want to buy a couple of things from the supermarket to prepare for a party.
+After scanning all the items the cashier wants to give you the total price, but
+the machine is broken! Let's write her a function that does it for her
+instead!
+
+1. Create an object named `cartForParty` with five properties. Each property
+   should be a grocery item (like `beers` or `chips`) and hold a number value
+   (like `1.75` or `0.99`).
+   const cartForParty ={
+
+   }
+
+2. Complete the function called `calculateTotalPrice`.
+
+   - It takes one parameter: an object that contains properties that only contain
+     number values.
+   - Loop through the object and add all the number values together.
+   - Return a string: "Total: €`amount`".
+
+3. Complete the unit test functions and verify that all is working as expected.
+-----------------------------------------------------------------------------*/
+const cartForParty = {
+  beers  : 1.75,
+  chips :  0.99,
+  chikenwings :  2.4,
+  water :  2,
+  bread :  1.5,
+};
```

Comment:
Nice, beers. What is your favourite? 😄

Small tip: there is too much spacing. 

```suggestion
const cartForParty = {
  beers: 1.75,
  chips:  0.99,
  chikenwings:  2.4,
  water:  2,
  bread:  1.5,
};
```